src/command.c

```diff
@@ -161,6 +165,13 @@ int command_process(target_s *const t, char *const cmd_buffer)
 			return !cmd->handler(t, argc, argv);
 	}
 
+#ifdef PLATFORM_HAS_CUSTOM_COMMANDS
+	for (const command_s *cmd = platform_cmd_list; cmd->cmd; ++cmd) {
+		if ((argc == 0) || !strncmp(argv[0], cmd->cmd, strlen(argv[0])))
```

Comment:
The `argc == 0` term here can be dropped as the only reason this exists in the block above is to handle `monitor` (no arguments) being given so that the version command fires to handle that.

Reply:
Removed.